### DISCUSSION TITLE
Terms: IsAvailableForTagging and LocalProperties

### DISCUSSION THREAD

**Discussion by mappinger** [pnp/pnpcore#1358] [General] [answered]
Hi,

I'm migrating some old code from PnP-Sites-Core and have some problems with following two "features":

- disable term from termset for tagging (Term.IsAvailableForTagging)
- read localproperties from term (Term.LocalProperties)

Am I missing the equivalent properties in PnP Core? Or are those properties no longer available?

Thanks and best regards,
Michael

**Comment by mappinger** [comment]
A bit more context: the code mirrors the structure of an organisation from a SAP export to the term store (therefore I need to store the id from SAP in localproperties and have to exclude old organisation units from tagging).

**Comment by jansenbe** [comment, marked as answer]
@mappinger : under the covers we're using Microsoft Graph for taxonomy features (https://learn.microsoft.com/en-us/graph/api/resources/termstore-term?view=graph-rest-1.0) and there's no support for either IsAvailableForTagging and LocalProperties. There is a generic `Properties` option on a term (see https://pnp.github.io/pnpcore/using-the-sdk/taxonomy-advanced.html#working-with-term-set-properties), maybe that can help in your case (it matches to the old shared properties)? Regarding term tagging, that's not available anymore in the modern term store, I would recommend moving away from that feature if possible.

**Comment by mappinger** [comment]
Thank you for your answer! Regarding the tagging, I meant the "deprecate/enable term" feature (which I still see in the modern term store manager). But this is also not available via Graph API, only via SharePoint Rest API. I've created an idea if anyone has the same requirements: https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/support-term-isdeprecated-property-in-graphapi/idi-p/4037200
